src/polkadot_rpc_wrapper.py

```diff
@@ -109,33 +111,39 @@ def find_validator_address(self):
             for k in keys.values():
                 # Some chains uses multiple keys. Before checking if it exist on the node they need to be concatenated removing preceding '0x'.
                 session_key += k[2:]
-            if self.has_session_key(session_key):
-                return {"validator": validator[0], "session_key": session_key}
-        return False
+            has_session_key, error = self.has_session_key(session_key)
+            if error:
+                return False, error
+            if has_session_key:
+                return {"validator": validator[0], "session_key": session_key}, ""
+        return False, ""
 
     def is_validating_next_era(self, address):
```

Comment:
Add return types for all functions?

You did for some, but not for all :man_shrugging: